### PR TITLE
Updated internationalisation : fr-fr

### PR DESCRIPTION
**Pull request by tostt**
# Comprehensive Summary of your change

Edited file `fr-fr-lang.json`:
- Translated strings into French

Due to the nature of the changes, no build or test of the application was performed.

Please indicate you've done the following:
- [X] Well Written Title and Summary of the PR
- [X] Label the PR as needed. "release-note/ignore-for-release, release-note/new-feature, release-note/update, release-note/enhancement, release-note/community, release-note/breaking-change, release-note/docs, release-note/infra, release-note/deprecation"
- [X] Accepted the DCO. Commits without the DCO will delay acceptance.
- [X] Made sure tests are passing and test coverage is added if needed.
- [X] Considered the docs impact and opened a new docs issue or PR with docs changes if needed in [website repository](https://github.com/goharbor/website).

### CODE REVIEW

**Review by Vad1mo — APPROVED** [review]

File: src/portal/src/i18n/lang/fr-fr-lang.json [hunk 1]
```diff
@@ -248,12 +248,12 @@
         "INLINE_HELP_PUBLIC": "Lorsqu'un projet est mis en public, n'importe qui a l'autorisation de lire les dépôts sous ce projet, et l'utilisateur n'a pas besoin d'exécuter \"docker login\" avant de prendre des images de ce projet.",
         "OF": "sur",
         "COUNT_QUOTA": "Quota de nombre",
-        "STORAGE_QUOTA": "Project quota limits",
+        "STORAGE_QUOTA": "Quota de projet",
```
Comment (flbla) [inline src/portal/src/i18n/lang/fr-fr-lang.json:251]:
```suggestion
        "STORAGE_QUOTA": "Quota du projet",
```

File: src/portal/src/i18n/lang/fr-fr-lang.json [hunk 2]
```diff
@@ -383,46 +383,46 @@
...
-        "REPLICATION": "Replication",
-        "REPLICATION_ADAPTER": "Replication Adapter",
-        "REGISTRY": "Registry",
-        "SCAN_ALL": "Scan All",
-        "SYSTEM_VOLUMES": "System Volumes",
-        "GARBAGE_COLLECTION": "Garbage Collection",
-        "PURGE_AUDIT": "Purge Audit",
+        "UPDATE": "Mettre à jour",
+        "AUDIT_LOG": "Log d'audit",
+        "PREHEAT_INSTANCE": "Préchauffer l'instance",
```
Comment (flbla) [inline src/portal/src/i18n/lang/fr-fr-lang.json:388]:
```suggestion
        "PREHEAT_INSTANCE": "Précharger l'instance",
```

File: src/portal/src/i18n/lang/fr-fr-lang.json [hunk 3]
```diff
@@ -383,46 +383,46 @@
...
         "SCANNER": "Scanner",
         "LABEL": "Label",
-        "EXPORT_CVE": "Export CVE",
-        "SECURITY_HUB": "Security Hub",
-        "CATALOG": "Catalog",
-        "METADATA": "Project Metadata",
-        "REPOSITORY": "Repository",
-        "ARTIFACT": "Artifact",
+        "EXPORT_CVE": "Exporter CVE",
+        "SECURITY_HUB": "Tableau de bord sécurité",
```
Comment (flbla) [inline src/portal/src/i18n/lang/fr-fr-lang.json:403]:
```suggestion
        "SECURITY_HUB": "Tableau de bord de sécurité",
```

File: src/portal/src/i18n/lang/fr-fr-lang.json [hunk 4]
```diff
@@ -383,46 +383,46 @@
...
         "TAG": "Tag",
-        "ACCESSORY": "Accessory",
-        "ARTIFACT_ADDITION": "Artifact Addition",
-        "ARTIFACT_LABEL": "Artifact Label",
-        "PREHEAT_POLICY": "Preheat Policy",
-        "IMMUTABLE_TAG": "Immutable Tag",
+        "ACCESSORY": "Accessoire",
+        "ARTIFACT_ADDITION": "Artefact Addition",
+        "ARTIFACT_LABEL": "Label d'artefact",
+        "PREHEAT_POLICY": "Politique de préchauffage",
```
Comment (flbla) [inline src/portal/src/i18n/lang/fr-fr-lang.json:413]:
```suggestion
        "PREHEAT_POLICY": "Politique de préchargement",
```

File: src/portal/src/i18n/lang/fr-fr-lang.json (hunk 2 above)
Comment (tostt) [reply src/portal/src/i18n/lang/fr-fr-lang.json:388]: I use the expression "préchauffage" to keep the translation consistent with other instances.

File: src/portal/src/i18n/lang/fr-fr-lang.json (hunk 1 above)
Comment (tostt) [reply src/portal/src/i18n/lang/fr-fr-lang.json]: Thank you for your suggestion.

File: src/portal/src/i18n/lang/fr-fr-lang.json (hunk 2 above)
Comment (tostt) [reply src/portal/src/i18n/lang/fr-fr-lang.json:388]: Thank you for your suggestion.

File: src/portal/src/i18n/lang/fr-fr-lang.json (hunk 3 above)
Comment (tostt) [reply src/portal/src/i18n/lang/fr-fr-lang.json]: Thank you for your suggestion.

File: src/portal/src/i18n/lang/fr-fr-lang.json [hunk 5]
```diff
@@ -383,46 +383,46 @@
...
+        "ARTIFACT": "Artefact",
         "SCAN": "Scan",
         "TAG": "Tag",
-        "ACCESSORY": "Accessory",
-        "ARTIFACT_ADDITION": "Artifact Addition",
-        "ARTIFACT_LABEL": "Artifact Label",
-        "PREHEAT_POLICY": "Preheat Policy",
-        "IMMUTABLE_TAG": "Immutable Tag",
+        "ACCESSORY": "Accessoire",
+        "ARTIFACT_ADDITION": "Artefact Addition",
```
Comment (thcdrt) [inline src/portal/src/i18n/lang/fr-fr-lang.json:411]: I think we should translate "addition" too but I don't understand the mean of this

File: src/portal/src/i18n/lang/fr-fr-lang.json [hunk 6]
```diff
@@ -383,46 +383,46 @@
...
-        "NEXT": "Next",
-        "FINISH": "Finish",
-        "BASIC_INFO": "Basic Information",
-        "SELECT_PERMISSIONS": "Select Permissions",
-        "SELECT_SYSTEM_PERMISSIONS": "Select System Permissions",
-        "SELECT_PROJECT_PERMISSIONS": "Select Project Permissions",
-        "SYSTEM_PERMISSIONS": "System Permissions"
+        "BACK": "Retour",
+        "NEXT": "Suivant",
+        "FINISH": "Fin",
```
Comment (thcdrt) [inline src/portal/src/i18n/lang/fr-fr-lang.json:420]:
```suggestion
        "FINISH": "Finir",
```
? Or "Terminer"

File: src/portal/src/i18n/lang/fr-fr-lang.json [hunk 7]
```diff
@@ -383,46 +383,46 @@
...
+        "BASIC_INFO": "Informations de base",
+        "SELECT_PERMISSIONS": "Selectionnez les permissions",
+        "SELECT_SYSTEM_PERMISSIONS": "Selectionnez les permissions système",
```
Comment (thcdrt) [inline src/portal/src/i18n/lang/fr-fr-lang.json:423]: Should we conjugate this or should it be infinitive ?

File: src/portal/src/i18n/lang/fr-fr-lang.json [hunk 8]
```diff
@@ -705,11 +705,11 @@
         "BANDWIDTH_TOOLTIP": "Set the maximum network bandwidth for each replication worker. Please pay attention to the number of concurrent executions (max. {{max_job_workers}}). For unlimited bandwidth, please enter -1.",
         "UNLIMITED": "Illimitée",
         "UNREACHABLE_SOURCE_REGISTRY": "Échec de connexion au registre source. Veuillez vérifier que le registre source est disponible avant d'éditer cette règle: {{error}}",
-        "CRON_ERROR_TIP": "The 1st field of the cron string must be 0 and the 2nd filed can not be \"*\"",
+        "CRON_ERROR_TIP": "Le 1er champ de la chaîne cron doit être 0 et le 2ème champ ne peut être \"*\"",
```
Comment (thcdrt) [inline src/portal/src/i18n/lang/fr-fr-lang.json:708]:
```suggestion
        "CRON_ERROR_TIP": "Le 1er champ de la chaîne cron doit être 0 et le 2ème champ ne peut pas être \"*\"",
```

File: src/portal/src/i18n/lang/fr-fr-lang.json [hunk 9]
```diff
@@ -1855,9 +1855,9 @@
         "SCHEDULE_RESUME_BTN_INFO": "REPRENDRE — Reprend les files d'attente de tâches à exécuter.",
         "WORKER_FREE_BTN_INFO": "Arrête les tâches en cours pour libérer le worker",
         "CRON": "Cron",
-        "WAITING_TOO_LONG_1": "Certain jobs have been pending for execution for over 24 hours. Please check the job service ",
-        "WAITING_TOO_LONG_2": "dashboard.",
-        "WAITING_TOO_LONG_3": "For more details, please refer to the ",
+        "WAITING_TOO_LONG_1": "Certaines tâches sont en attente d'exécution depuis plus de 24 heures. Veuillez vérifier le tableau de bord ",
+        "WAITING_TOO_LONG_2": "du service des tâches.",
```
Comment (thcdrt) [inline src/portal/src/i18n/lang/fr-fr-lang.json:1859]:
```suggestion
        "WAITING_TOO_LONG_2": "tableau de bord.",
```

**Review by thcdrt — CHANGES_REQUESTED** [review]
Hello,

Thank you for your contribution, here are some suggestions

File: src/portal/src/i18n/lang/fr-fr-lang.json (hunk 7 above)
Comment (thcdrt) [reply src/portal/src/i18n/lang/fr-fr-lang.json:423]: Same for 2 lines around

File: src/portal/src/i18n/lang/fr-fr-lang.json [hunk 10]
```diff
@@ -383,46 +383,46 @@
...
+        "PREHEAT_INSTANCE": "Préchauffer l'instance",
+        "PROJECT": "Projet",
+        "REPLICATION_POLICY": "Politique de réplication",
+        "REPLICATION": "Réplication",
+        "REPLICATION_ADAPTER": "Adapter de réplication",
+        "REGISTRY": "Registre",
+        "SCAN_ALL": "Scanner tout",
+        "SYSTEM_VOLUMES": "Volumes système",
+        "GARBAGE_COLLECTION": "Purge",
+        "PURGE_AUDIT": "Audit de purge",
```
[the remark this answers is not in the document]
Comment (tostt) [reply src/portal/src/i18n/lang/fr-fr-lang.json]: I changed it to "Purger l'audit".

File: src/portal/src/i18n/lang/fr-fr-lang.json [hunk 11]
```diff
@@ -383,46 +383,46 @@
...
         "SCANNER": "Scanner",
         "LABEL": "Label",
-        "EXPORT_CVE": "Export CVE",
-        "SECURITY_HUB": "Security Hub",
-        "CATALOG": "Catalog",
-        "METADATA": "Project Metadata",
-        "REPOSITORY": "Repository",
-        "ARTIFACT": "Artifact",
+        "EXPORT_CVE": "Exporter CVE",
+        "SECURITY_HUB": "Tableau de bord de sécurité",
```
[the remark this answers is not in the document]
Comment (tostt) [reply src/portal/src/i18n/lang/fr-fr-lang.json]: Changed.

File: src/portal/src/i18n/lang/fr-fr-lang.json (hunk 5 above)
Comment (tostt) [reply src/portal/src/i18n/lang/fr-fr-lang.json:411]: I don't know what this is, so I didn't change it.

**Review by thcdrt — APPROVED** [review]

**Comment by wy65701436** [comment]
thanks @tostt @thcdrt for your contributions, I will merge this Pr and ship it with v2.11

**Pull request merged**